### PR TITLE
Meta description

### PR DESCRIPTION
**Pull request by luciferankon**

### CODE REVIEW

File: getting_started/installing-gauge.rst [hunk 1]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+   :description: A simple guide to install Gauge on your system depending on your os...
```
Comment (zabil) [inline getting_started/installing-gauge.rst]: A simple guide for installing Gauge on Windows, macOS, Ubuntu/Linux, FreeBSD

File: getting_started/view-a-report.rst [hunk 2]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: A quick overview of how to view the report of last test run. It guides you through how to view report of last execution on different os, in different languages, and on different ides...
```
Comment (zabil) [inline getting_started/view-a-report.rst]: A quick overview of how to view test reports. It guides you through how to view report of last execution on different os, in different languages, and on different ides..

File: howto/ci_cd/azure.rst [hunk 3]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: This is tutorial on how to integrate Gauge with Azure
```
Comment (zabil) [inline howto/ci_cd/azure.rst]: A tutorial on how to integrate Gauge with Azure

File: howto/ci_cd/circle.rst [hunk 4]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: This is tutorial on how to integrate Gauge with CircleCI
```
Comment (zabil) [inline howto/ci_cd/circle.rst]: A tutorial on how to integrate Gauge with CircleCI

File: howto/ci_cd/docker.rst [hunk 5]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: This is tutorial on how to install Gauge in Docker
```
Comment (zabil) [inline howto/ci_cd/docker.rst]: A tutorial on how to install Gauge in Docker

File: howto/ci_cd/gocd.rst [hunk 6]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: This is tutorial on how to integrate Gauge with GoCD
```
Comment (zabil) [inline howto/ci_cd/gocd.rst]: A tutorial on how to integrate Gauge with GoCD

File: howto/ci_cd/jenkins.rst [hunk 7]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: This is tutorial on how to integrate Gauge with Jenkins
```
Comment (zabil) [inline howto/ci_cd/jenkins.rst]: A on how to integrate Gauge with Jenkins

File: howto/ci_cd/teamcity.rst [hunk 8]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: This is tutorial on how to integrate Gauge with TeamCity
```
Comment (zabil) [inline howto/ci_cd/teamcity.rst]: A tutorial on how to integrate Gauge with TeamCity

File: howto/ci_cd/travis.rst [hunk 9]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: This is tutorial on how to integrate Gauge with Travis CI
```
Comment (zabil) [inline howto/ci_cd/travis.rst]: A tutorial on how to integrate Gauge with Travis CI

File: troubleshooting.rst [hunk 10]
```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: Some common problems and the way to troubleshoot them.
```
Comment (zabil) [inline troubleshooting.rst]: Some common problems and ways to troubleshoot them.

**Comment by luciferankon** [comment]
@gaugebot recheck

**Comment by luciferankon** [comment]
@gaugebot recheck

**Pull request merged**